README.md

```diff
@@ -0,0 +1,2 @@
+#### Example usage
+`docker-sigprofiler -i wkdir/data/DATA.txt -o wkdir/result -it table -r GRCh37 -ms 2 -ts 3 -n 3`
```

Comment:
I think the documentation is a bit too short.

Could you produce something similar to the one in telomerecat: https://github.com/cancerit/telomerecat/tree/feature/explore_to_py3

If `docker-sigprofiler -h` prints out enough usage for user to use your tool, you don't have to have a `Example usage` section in your readme, otherwise you'll have to update it everytime you update your tool usage.